m4/ax_code_coverage.m4

```diff
@@ -0,0 +1,272 @@
+# ===========================================================================
```

Comment:
Is this file supposed to be symlinked like most of the other files I see in my m4 directory, rather than a copy checked in to git?

Reply:
Ideally we shouldn't need to include a copy in our repository but the version of this file in the GitHub Ubuntu environment is too old, and has a problem that prevents `make distcheck` from completing.